README.md

```diff
@@ -1,33 +1,63 @@
 
-This repository contains a script to build a GNU toolchain cross compiler for Windows-on-arm64. It is a **work in progress**. This allows binaries that can execute on Windows-on-arm64 to be built on Linux. I have only tested this script on Ubuntu/WSL.
+This repository contains scripts for building a `x86_64-pc-linux-gnu` host, `aarch64-w64-mingw32` target GNU toolchain cross-compiler for Windows on Arm64. It is a **work in progress** in the stage of upstreaming the necessary changes to the corresponding repositories. This toolchain contains binaries that can be executed on Windows on Arm64 to be built on Linux. The scripts are being actively tested on the default Ubuntu 22.04 GitHub Actions runners and Ubuntu 22.04 in WSL.
 
-# Known issues:
-This toolchain is not yet ready for real-world use. Problems and missing parts are listed in the [issues](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/issues).
+# Known Issues
 
-# Install build dependencies:
+This toolchain is not yet ready for real-world use. Problems and missing parts are listed in the [issues](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/issues). Please, report all issues, even if the issue is, e.g., GCC-specific, here.
 
-sudo apt-get install build-essential binutils-for-build texinfo bison flex zlib1g-dev libgmp-dev dejagnu libmpc-dev
+# Installing Build Dependencies
 
-# Build the cross-compiler
+The [main build script](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/blob/main/build.sh) installs its dependencies automatically when `RUN_BOOTSTRAP=1` environment variable is defined which is the default. To see what will be installed refer to [`.github/scripts/install-dependencies.sh`](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/blob/main/.github/scripts/install-dependencies.sh).
 
-Running build.sh will build the cross compiler. It installs it into ~/cross. The build script does multiple things.
- - Downloads dependencies
- - Clones several dependencies 
-    - modified binutils from https://github.com/Windows-on-ARM-Experiments/binutils-woarm64.git
-    - modified gcc from https://github.com/Windows-on-ARM-Experiments/gcc-woarm64
-    - modified mingw from https://github.com/Windows-on-ARM-Experiments/mingw-woarm64
- - Builds and installs: binutils, mingw, gcc and libgcc.
+# Building the Cross-Compiler
 
-# Using the cross compiler
+Running [`./build.sh`](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/blob/main/build.sh) on Ubuntu 22.04 in the root repository folder will build the cross-compiler and install it into `~/cross-aarch64-w64-mingw32-msvcrt` folder by default.
```

Comment:
For simplicity it would be good add step with cloning and entering the folder.

Reply:
Added steps with clonning.